test/rating.js

```diff
@@ -16,40 +17,13 @@ describe('A track in Google Music', function () {
   });
   browserUtils.execute(function playViaApi () {
     window.googleMusic.playback.playPause();
+    window.googleMusic.playback.toggleRepeat(window.GoogleMusic.Playback.SINGLE_REPEAT);
```

Comment:
No, I extracted, formalized, and tested the code from `radiant-player-mac`. At some point, I got burned out and didn't make everything consistent. See examples here:

https://github.com/gmusic-utils/gmusic.js/issues